toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -267,12 +267,12 @@ make-4.2.1-5.cm1.aarch64.rpm
 make-debuginfo-4.2.1-5.cm1.aarch64.rpm
 mariner-check-macros-1.0-3.cm1.noarch.rpm
 mariner-release-1.0-19.cm1.noarch.rpm
-mariner-repos-1.0-13.cm1.noarch.rpm
-mariner-repos-extras-1.0-13.cm1.noarch.rpm
-mariner-repos-extras-preview-1.0-13.cm1.noarch.rpm
-mariner-repos-preview-1.0-13.cm1.noarch.rpm
-mariner-repos-ui-1.0-13.cm1.noarch.rpm
-mariner-repos-ui-preview-1.0-13.cm1.noarch.rpm
+mariner-repos-1.0-14.cm1.noarch.rpm
+mariner-repos-extras-1.0-14.cm1.noarch.rpm
+mariner-repos-extras-preview-1.0-14.cm1.noarch.rpm
+mariner-repos-preview-1.0-14.cm1.noarch.rpm
+mariner-repos-ui-1.0-14.cm1.noarch.rpm
+mariner-repos-ui-preview-1.0-14.cm1.noarch.rpm
```

Comment:
I think you need to add the two new subpackages here - we list all packages built in the toolchain in this file, if I'm not mistaken.

Reply:
You'd also need to then also update `toolkit/resources/manifests/package/update_manifests.sh`.

Reply:
Interesting...why would we do that?  Seems like that causes a circular dependency between the toolchain and the downstream repos.

Reply:
Updated.

Reply:
Thank you, approved. I didn't get the comment about circular dependencies, though. If you have time and think there's a bug here we should address, let's talk offline.